MobileWallet/Screens/Scan/ScanViewController.swift

```diff
@@ -0,0 +1,209 @@
+
+	2. Redistributions in binary form must reproduce the above
+	copyright notice, this list of conditions and the following disclaimer in the
+	documentation and/or other materials provided with the distribution.
+
+	3. Neither the name of the copyright holder nor the names of
+	its contributors may be used to endorse or promote products
+	derived from this software without specific prior written permission.
+
+	THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+	CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+	INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+	OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+	DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+	CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+	SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+	NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+	LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+	HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+	CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+	OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+	SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+import UIKit
+import AVFoundation
+
+protocol ScanViewControllerDelegate {
+    func found(code: String)
+}
+
+class ScanViewController: UIViewController {
+
+    // MARK: - Variables and constants
+    var captureSession: AVCaptureSession!
+    var previewLayer: AVCaptureVideoPreviewLayer!
+    var delegate: ScanViewControllerDelegate?
+
+    let widthRectanglePath: CGFloat = CGFloat(276)
+    let heightRectanglePath: CGFloat = CGFloat(259)
+    let distanceToTitleLabel: CGFloat = CGFloat(80)
+
+    // MARK: - Outlets
+    @IBOutlet weak var backButton: UIButton!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var middleView: UIView!
+    @IBOutlet weak var topLeftWhiteView: UIView!
+    @IBOutlet weak var topRightWhiteView: UIView!
+    @IBOutlet weak var bottomLeftWhiteView: UIView!
+    @IBOutlet weak var bottomRightWhiteView: UIView!
+
+    // MARK: - Override functions
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        customizeScanner()
+        customizeViews()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        if (captureSession?.isRunning == false) {
+            captureSession.startRunning()
+        }
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+
+        if (captureSession?.isRunning == true) {
+            captureSession.stopRunning()
+        }
+    }
+
+    override var prefersStatusBarHidden: Bool {
+        return true
+    }
+
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return .portrait
+    }
+
+    // MARK: - Private functions
+    private func customizeViews() {
+        topLeftWhiteView.addTopBorder(with: .white, andWidth: 11)
```

Comment:
Move the color into the `Theme.swift` file. Don't want to have any hardcoded colors in the app so it's easier to tweak designs globally later especially when we implement dark mode.

Reply:
Done